PiicoDev_Unified.py

```diff
@@ -6,6 +6,14 @@
 compat_ind = 1
```

Comment:
This and compat_ind be in SCREAMING_SNAKE_CASE if they're being defined as constant (although since they're referred to by other modules this may be difficult to change unfortunately)